src/main/java/insat/company/platform/web/rest/AccountResource.java

```diff
@@ -160,6 +160,23 @@ public void requestPasswordReset(@RequestBody String mail) {
        );
     }
 
+
+    /**
+     * POST   /account/invite : Send an email to invite a user
+     * @param mail the mail of the user
+     * @throws EmailAlreadyUsedException 400 (Bad Request) if the new Email is already Used
+     */
+    @PostMapping(path = "/account/invite")
+    @Timed
+    public void requestInviteEmail(@RequestBody String mail) {
+
+        if (userService.VerifyEmail(mail).isPresent()) {
+            throw new EmailAlreadyUsedException();
```

Comment:
you should not throw an expection in the controller, you should return a response entity